style.css

```diff
@@ -455,8 +461,344 @@ ul.card-item {
+    text-align: center;
+    color: #172b4d;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    margin: 24px -320px;
+  }
+
+  .dis {
+    margin-left: 50px;
+  }
+
+  .card-sections {
+    align-items: center;
+    margin: 10px;
+  }
+
+  .card-sections h1 {
+    height: 15px;
+    padding-left: 37%;
+    font-family: 'inter', sans-serif !important;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 35px;
+    line-height: 52px;
+    display: flex;
+    color: #172b4d;
+  }
+
+  .indicator {
+    display: none;
+  }
+
+  .button button {
+    width: 79px;
+    height: 16px;
+    left: 12px;
+    top: 8px;
+    font-family: 'inter', sans-serif !important;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #172b4d;
+    padding-left: 20px;
+  }
+
+  .work-section {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    padding-top: 80px;
+  }
+
+  .items ul {
+    padding-left: 35px;
+    border-radius: 18px;
+  }
+
+  .button-cont {
+    margin-left: 98px;
+  }
+
+  .items ul li {
+    font-size: 18px;
+  }
+
+  .card-lists h3 {
+    width: 337px;
+    height: 57px;
+    padding-left: 16px;
+    font-family: 'Poppins', sans-serif !important;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 26px;
+    line-height: 32px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #172b4d;
+    flex: none;
+  }
+
+  .container-card {
+    margin: 20px;
+  }
+
+  .flex-container {
+    margin-left: -50px;
+  }
+
+  .about-me h1 {
+    width: 447px;
+    height: 48px;
+    padding-left: 550px;
+    margin-top: -115px;
+    font-family: 'inter', sans-serif !important;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+
+    /* or 130% */
+
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #172b4d;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .about-me p {
+    width: 770px;
+    height: 96px;
+    font-family: 'inter', sans-serif !important;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    color: #42526e;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    margin: 24px 260px;
+  }
+
+  .bot-img {
+    padding-left: 0;
+    margin-top: -100px;
+  }
+
+  .pic3 {
+    margin-left: 87%;
+  }
+
+  .botton-con {
+    margin-left: 22%;
+    margin-top: 25px;
+    padding: 40px;
+  }
+
+  .last-section {
+    display: flex;
+    margin-left: 8%;
+  }
+
+  .details p {
+    width: 463px;
+    height: 256px;
+    left: 0;
+    top: 0;
+    font-family: 'inter', sans-serif !important;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+
+    /* margin: 24px 0; */
```

Comment:
- [ ] To make your code clean...
Please, clear your project from unused commented codes!

Reply:
done

Reply:
done